vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt

```diff
@@ -267,7 +267,8 @@ class HomeActivityViewModel @AssistedInject constructor(
 //        }
 
         when {
-            !vectorPreferences.areThreadMessagesEnabled() && !vectorPreferences.wasThreadFlagChangedManually() -> {
+            // Tchap : disable automatic thread migration
+            !vectorPreferences.areThreadMessagesEnabled() && !vectorPreferences.wasThreadFlagChangedManually() && vectorFeatures.tchapIsThreadEnabled() -> {
```

Comment:
Thread migration is called multiple times in the app. You should disable this migration.